<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogin</class>
 <widget class="QDialog" name="frmLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ввод логина и пароля</string>
  </property>
  <widget class="QLineEdit" name="edtLogin">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>36</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblImg">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>155</width>
     <height>147</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/lock.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblUserPassword">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>78</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPassword">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>76</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>8</y>
     <width>387</width>
     <height>173</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="btnEnter">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>122</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>122</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblUserName">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>38</y>
     <width>42</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>edtLogin</zorder>
  <zorder>lblImg</zorder>
  <zorder>lblUserPassword</zorder>
  <zorder>edtPassword</zorder>
  <zorder>lblUserName</zorder>
 </widget>
 <tabstops>
  <tabstop>edtLogin</tabstop>
  <tabstop>edtPassword</tabstop>
  <tabstop>btnEnter</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
